serverpool/src/main/java/com/ats/serverpool/network/tcp/client/TCPClient.java

```diff
@@ -0,0 +1,41 @@
+package main.java.com.ats.serverpool.network.tcp.client;
+
+import java.io.*;
+import java.net.*;
+import com.ats.serverpool.Peer;
+
+class TCPClient {
+    private Socket socket;
+    private Peer peer;
+
+    public TCPClient(Peer peer) {
+        this.peer = peer;
+
+        try {
+            this.socket = new Socket(peer.getIp(), peer.getPort());
+        } catch(SocketException e) {
+            System.out.println("Error initalizing Socket");
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public void sendMsg(String msg) {
```

Comment:
I'm not sure how the tcp messages will be structured, but wouldnt you want to pass a Message.java class and use the toBytes() method to get the message in Bytes?

Reply:
@aleksajovanovic I'll merge so you can continue with the test. This is a minor change and can be adjusted later if applicable.